src/Configuration.php

```diff
@@ -41,47 +41,38 @@ public static function fromArray(array $arr)
     }
 
     /**
-     * @param $layers
-     * @param $ruleset
-     * @param $paths
-     * @param $exclude_files
+     * @param ConfigurationLayer[] $layers
+     * @param ConfigurationRuleset $ruleset
+     * @param array                $paths
+     * @param array                $excludeFiles
```

Comment:
I would now drop the docblock. It doesn't add any value.

Reply:
fixed;  replaced `array` with `string[]`